interfaces/Portalicious/.prettierrc.yml

```diff
@@ -11,6 +11,7 @@ plugins:
   - '@prettier/plugin-xml'
 tailwindAttributes:
   - styleClass
+  - icon
```

Comment:
Isn't it possible to have 1 central `tailwind.config.json` or something?

One that both Prettier AND the VSCode-extension can/should use?

Reply:
Not that I'm aware of.

Reply:
Or better, we do have a central tailwind.config but that doesn't deal with these attributes.